### PR TITLE
Update video.yml

### PR DESCRIPTION
**Pull request by notsoocool**
- url: https://www.youtube.com/watch?v=aOIzz64E-bE&t=76s
  title: FOSSASIA SUMMIT
  img: yajushvy.png
  author_name: Yajush Vyas

### CODE REVIEW

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please upload a new image in the img/video folder

**Review by isuruAb — COMMENTED** [review]
please upload image to img/video folder

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Squash your commits as well :+1:

**Comment by Yajushvyas** [comment]
cbgfh

File: _data/video.yml [hunk 1]
```diff
@@ -32,3 +32,8 @@
   img: FOSSASIA.jpeg
   author_name: Naveen Rajan
   
+- url: https://www.youtube.com/watch?v=aOIzz64E-bE&t=76s
+  title: FOSSASIA SUMMIT
+  img: yajushvyas.png
```
Comment (notsoocool) [inline _data/video.yml:37]: yajushvy

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Please upload the image to `img/video` folder

**Pull request closed without merging**